playbooks/common/openshift-master/config.yml

```diff
@@ -337,12 +361,54 @@
     with_items: openshift_master_named_certificates
     when: named_certs_specified | bool
 
-- name: Configure master instances
-  hosts: oo_masters_to_config
+- name: Configure first master instance
```

Comment:
I would propose we make the goal a consistent "protect currently installed unless upgrading", rather than "keep cluster in sync", and make the latter the job of a specific playbook for updating the cluster to latest version in the current stream. Seem reasonable?